backends-velox/src/test/scala/io/glutenproject/execution/VeloxFunctionsValidateSuite.scala

```diff
@@ -383,4 +383,26 @@ class VeloxFunctionsValidateSuite extends VeloxWholeStageTransformerSuite {
       checkOperatorMatch[ProjectExecTransformer]
     }
   }
+
+  test("Test unbase64 function"){
+
+    withTempPath {
+      path =>
+        Seq(
+          "U3BhcmsgU1FM",
+          null
+        )
+          .toDF("e")
+          .write
+          .parquet(path.getCanonicalPath)
+
+        spark.read.parquet(path.getCanonicalPath).createOrReplaceTempView("base64_tbl")
+
+        runQueryAndCompare("select unbase64(e) from base64_tbl") {
+          checkOperatorMatch[ProjectExecTransformer]
+        }
+    }
```

Comment:
Why we need to write/read file for testing this function?

Reply:
Looks like we don't neet to write and read file test , just write a few fixed-value tests? Thanks!